src/main/java/io/ballerina/plugins/idea/sdk/OSUtils.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.ballerina.plugins.idea.sdk;
+
+import java.util.Locale;
+
+/**
+ * Utility functions used by launcher.
+ *
+ * @since 2.0.0
+ */
+public class OSUtils {
```

Comment:
Better to keep this as a generic util, instead of keeping under the `sdk` package